{"name": "Невозможность авторизоваться курьером с некорректным значением поля", "status": "passed", "steps": [{"name": "Отправка запроса на авторизацию курьера с некорректными данными", "status": "passed", "start": 1710628107234, "stop": 1710628108017}], "parameters": [{"name": "key", "value": "'login'"}, {"name": "wrong_field", "value": "'wrong@login'"}], "start": 1710628107234, "stop": 1710628108020, "uuid": "ae4c9ba6-6e7e-4943-a685-a76adb2f61a5", "historyId": "c8214069ae2a0f58db953d0c4765a8f8", "testCaseId": "95bfda5ea90692b56bb543bf0b3a4bfb", "fullName": "tests.test_login_courier.TestLoginCourier#test_log_in_courier_with_wrong_field_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-L83GKGS"}, {"name": "thread", "value": "10608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}